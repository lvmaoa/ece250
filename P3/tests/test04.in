load
s carmen
s zebra
spellcheck zebra
spellcheck carmen
exit

