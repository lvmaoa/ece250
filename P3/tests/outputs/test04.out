success
found carmen
not found

correct
